### PR TITLE
Features/controllers

### PR DESCRIPTION
**Pull request by oguntayomicheal**
### In this PR, the following tasks were accomplished;
- Add routes and controller files for users
- Add users' view files
- Add routes and controller for post
- Add posts view files

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -2,7 +2,9 @@
 
 Rails.application.routes.draw do
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
+  resources :users do
+    resources :posts
+  end
```
Comment (rloterh) [inline config/routes.rb]: - [ ] Good job so far, however, we don't intend to expose unnecessary routes to users. Let's address this situation by keeping our routes in following manner:

```ruby
  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
```

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Mike, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 
🏁 🚦 🌴  🏃 🌳 🏡 🏘️ 🌲 

## Highlights 👍  
- Good job so far adding controllers and associated views and routes!
- Professional ReadMe!

## Required Changes ♻️
_Kindly check the inline comments under the review._

Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by amejid — APPROVED** [review]
Hi @oguntayomicheal, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Linter checks passing ✔️ 
- Correct workflow ✔️ 
- Routes fixed to expose only necessary endpoints ✔️ 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by oguntayomicheal** [comment]
Hi @amejid and @rloterh , thanks for the review

**Pull request merged**